apps/ledger-live-mobile/src/components/WebPTXPlayer/index.tsx

```diff
@@ -117,9 +123,29 @@ type Props = {
   manifest: LiveAppManifest;
   inputs?: Record<string, string | undefined>;
   disableHeader?: boolean;
+  config?:
+    | {
+        screen: ScreenName.ExchangeBuy | ScreenName.ExchangeSell;
+        navigator: NavigatorName.Exchange;
+        btnText: string;
+      }
+    | {
+        screen: ScreenName.Card;
+        navigator: NavigatorName.Card;
+        btnText: string;
+      };
 };
 
-export const WebPTXPlayer = ({ manifest, inputs, disableHeader }: Props) => {
+export const WebPTXPlayer = ({
+  manifest,
+  inputs,
+  disableHeader,
+  config = {
+    screen: ScreenName.ExchangeSell,
+    btnText: manifest.name,
+    navigator: NavigatorName.Exchange,
```

Comment:
Here i believe you should have something like above, `navigator: NavigatorName.Exchange | NavigatorName.Card;`

I am not sure why we call buy and sell exchange but card has it's own name? Shouldn't it be card as well?

Reply:
Basically, I believe Card should live under the `Exchange` navigator. Please correct me if i am wrong.

Reply:
This is the default value, so it doesn't make sense to add an 'or' condition here. I have set a default value, which follows the existing logic
On the other hand, Exchange has historically been a Buy and Sell app, so I'm not entirely convinced about using 'exchangeCard.' If we go that route, we'd also have to use names like 'exchangeSwap,' 'exchangeStake,' and so on.